cookbooks/mantle_convection_with_continents_in_annulus/modelR.prm

```diff
@@ -0,0 +1,209 @@
+# input file for model R of van der wiel et al, 2024.
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 1e9
+set Output directory                       = output-mantle-convection-annulus
+set World builder file                     = world.wb
+set CFL number                             = 0.7
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 10
+set Nonlinear solver tolerance             = 1e-4
+set Timing output frequency                = 10
+
+subsection Solver parameters
+    subsection Stokes solver parameters
+      set Number of cheap Stokes solver steps = 0
+      set Linear solver tolerance = 1e-6
+      set Maximum number of expensive Stokes solver steps = 5000
+   end
+end
+
+subsection Formulation
+   set Formulation = isentropic compression
+end
+
+subsection Compositional fields
+    set Number of fields = 2
+    set Names of fields = 1_mantle, 3_continent
+end
+
+subsection Nullspace removal
+  set Remove nullspace = net rotation
+end
+
+subsection Material model
+  set Model name = compositing
+  subsection Visco Plastic
+    set Thermal expansivities = 3e-5, 3e-5, 3e-5
```

Comment:
Add a comment here that `Thermal expansivities` and `Thermal diffusivities` are not actually used, because they are computed by the Latent heat model below. Or just delete the parameters from here (I dont know why they are here).